examples/react-native/index.js

```diff
@@ -3,19 +3,19 @@
  */
 
 import Config from 'react-native-config';
-import DashX from '@dashx/react-native';
 import { AppRegistry } from 'react-native';
 import App from './App';
 import { name as appName } from './app.json';
 
+import DashX from '@dashx/react-native';
+
 DashX.setLogLevel(0);
 
 DashX.setup({
   publicKey: Config.DASHX_PUBLIC_KEY,
   baseUri: 'http://api.dashx-staging.com/v1',
 });
 
-DashX.identify({ firstName: 'John', lastName: 'Doe' });
-DashX.track('Hello World');
+DashX.setIdentityToken(Config.DASHX_IDENTITY_TOKEN);
```

Comment:
Please revert this change. We don't want users thinking it's best practice to store an identity token in config.